discovery-provider/src/tasks/index_cache.py

```diff
@@ -9,12 +10,13 @@
 
 ######## HELPER FUNCTIONS ########
 def update_trending_cache(self, db, redis, time):
-    resp = generate_trending(db, time, None, 1000, 0)
-    resp_json = json.dumps(resp)
-    redis_key = f"trending-{time}"
-    # Cache value for 5 minutes
-    redis.set(redis_key, resp_json, 300)
-    logger.info(f"index_cache.py | Updated trending cache {redis_key}")
+    with db.scoped_session() as session:
```

Comment:
one kind of unrelated question here. for the trending, does it regenerate trending for genre for someone if they pass in a new app id?

Reply:
nope! we thought of this and made the redis cache decorator a bit smart - it ignores `app_name`. you can see this in `query_params.py:stringify_query_params`